mono/tests/Makefile.am

```diff
@@ -940,40 +946,26 @@ DISABLED_TESTS = \
 	$(PROFILE_DISABLED_TESTS) \
 	$(if $(AOT),$(AOT_DISABLED_TESTS)) \
 	$(if $(CI),$(CI_DISABLED_TESTS)) \
-	$(if $(CI_PR),$CI_PR_DISABLED_TESTS) \
+	$(if $(CI_PR),$(CI_PR_DISABLED_TESTS)) \
```

Comment:
given that CI_PR_DISABLED_TESTS wasn't actually used we can probably reenable appdomain-threadpool-unload.exe as well since it doesn't show up as failing anymore.

Reply:
hmm, I just found two occurrences of crashes in that test in the last 7 days and we have a bug tracking it: https://bugzilla.xamarin.com/show_bug.cgi?id=59976, so we should probably leave it disabled for now